Production/test/condorSub/dict_RunIIFall17MiniAODv2_qcd_pt.py

```diff
@@ -1,20 +1,33 @@
 flist = {
     "scenario": "Fall17",
     "samples": [
+        ['RunIIFall17MiniAODv2.QCD_Pt_80to120_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_80to120_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_120to170_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_170to300_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt_170to300_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_300to470_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_300to470_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_470to600_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_600to800_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt_600to800_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_800to1000_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_800to1000_TuneCP5_13TeV_pythia8_ext1'],
-        ['RunIIFall17MiniAODv2.QCD_Pt_1000to1400_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_1000to1400_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt_1000to1400_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_1400to1800_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt_1400to1800_TuneCP5_13TeV_pythia8_ext1'],
         ['RunIIFall17MiniAODv2.QCD_Pt_1800to2400_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_2400to3200_TuneCP5_13TeV_pythia8'],
         ['RunIIFall17MiniAODv2.QCD_Pt_3200toInf_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-15to20_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-20to30_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-30to50_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-50to80_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-80to120_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-120to170_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-170to300_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-300to470_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
+        ['RunIIFall17MiniAODv2.QCD_Pt-470to600_MuEnrichedPt5_TuneCP5_13TeV_pythia8'],
```

Comment:
are there higher-pT samples for MuEnriched?

Reply:
Yes, but some aren't complete and some aren't even being produced yet.

Reply:
What about cross sections?

Reply:
They are there now.